newIDE/app/src/AssetStore/index.js

```diff
@@ -614,27 +663,31 @@ export const AssetStore = React.forwardRef<Props, AssetStoreInterface>(
           <Line justifyContent="space-between" noMargin alignItems="center">
             {(!isOnHomePage || !!openedShopCategory) && (
               <>
-                <Column expand alignItems="flex-start" noMargin>
-                  <TextButton
-                    icon={<ChevronArrowLeft />}
-                    label={<Trans>Back</Trans>}
-                    onClick={async () => {
-                      const page = shopNavigationState.backToPreviousPage();
-                      const isUpdatingSearchtext = reApplySearchTextIfNeeded(
-                        page
-                      );
-                      if (isUpdatingSearchtext) {
-                        // Updating the search is not instant, so we cannot apply the scroll position
-                        // right away. We force a wait as there's no easy way to know when results are completely updated.
-                        await new Promise(resolve => setTimeout(resolve, 500));
-                        setScrollUpdateIsNeeded(page);
-                        applyBackScrollPosition(page); // We apply it manually, because the layout effect won't be called again.
-                      } else {
-                        setScrollUpdateIsNeeded(page);
-                      }
-                    }}
-                  />
-                </Column>
+                {shopNavigationState.isRootPage ? null : (
+                  <Column expand alignItems="flex-start" noMargin>
+                    <TextButton
+                      icon={<ChevronArrowLeft />}
+                      label={<Trans>Back</Trans>}
+                      onClick={async () => {
+                        const page = shopNavigationState.backToPreviousPage();
+                        const isUpdatingSearchtext = reApplySearchTextIfNeeded(
+                          page
+                        );
+                        if (isUpdatingSearchtext) {
+                          // Updating the search is not instant, so we cannot apply the scroll position
+                          // right away. We force a wait as there's no easy way to know when results are completely updated.
+                          await new Promise(resolve =>
+                            setTimeout(resolve, 500)
+                          );
```

Comment:
Let's take the opportunity to:
```suggestion
                          await delay(500);
```